### PR TITLE
Improve comments related to compression levels.

### PR DESCRIPTION
**Pull request by dhruba**

### CODE REVIEW

File: include/rocksdb/options.h [hunk 1]
```diff
@@ -381,9 +381,11 @@ struct ColumnFamilyOptions {
 
   // Compression algorithm that will be used for the bottommost level that
   // contain files. If level-compaction is used, this option will only affect
-  // levels after base level.
+  // levels after base level. This setting overides the compression settings
```
Comment (IslamAbdelRahman) [inline include/rocksdb/options.h]: nit: `overrides`

File: include/rocksdb/options.h [hunk 2]
```diff
@@ -381,9 +381,11 @@ struct ColumnFamilyOptions {
...
+  // configured via Options.compression but only for the bottommost level.
   //
-  // Default: kDisableCompressionOption (Disabled)
+  // Default: kDisableCompressionOption (Disabled). The means that the setting
```
Comment (IslamAbdelRahman) [inline include/rocksdb/options.h]: nit:
`The means` => `This means`
minor suggestion:
`the setting` => `the compression`

File: LICENSE [hunk 3]
```diff
@@ -1,5 +1,8 @@
 BSD License
 
+Copyright (c) 2016-present, Rockset, Inc.
+All rights reserved.
+---------------------------------------------------------------------
```
Comment (siying) [inline LICENSE]: @dhruba I think the PR still contains mostly unintended changes.

**Comment by siying** [comment]
@dhruba @veeve I think you must be using the branch for the pull request for something else. The pull request is "Improve comments related to compression levels", can you use another branch for this?

**Comment by dhruba** [comment]
I did not know that this was creating so many updates to the pull requests (automatically). Will close this one, create a new clean one in its place.

**Pull request closed without merging**